scripts/generatePackage.js

```diff
@@ -4,24 +4,21 @@ import sortPackageJson from 'sort-package-json';
 import prettier from 'prettier';
 import chalk from 'chalk';
 
-import templatePackageJson from './package-template/package.json';
```

Comment:
nit: I would rather keep this as normal imports instead because it will throw TS error in case someone move or rename something. Maybe even add it to scopes array as field then.

Reply:
Updated